specification/network/resource-manager/Microsoft.Network/stable/2019-12-01/expressRouteCircuit.json

```diff
@@ -1948,6 +1948,19 @@
         "modelAsString": true
       }
     },
+    "Ipv6CircuitConnectionConfig": {
+      "description": "IPv6 Circuit Connection properties for Global Reach.",
```

Comment:
Elsewhere it's written `global reach` (no caps). Please could you make this consistent?

Reply:
Done